config/caixa_config.go

```diff
@@ -0,0 +1,15 @@
+package config
+
+import "os"
+
+type CaixaConfig struct {
+	CaixaEnv               string
+	URLCaixaRegisterBoleto string
+}
+
+func getCaixaConfig() CaixaConfig {
```

Comment:
Opa, sem voltar o ponteiro, tu vai ter criado um objeto(struct), e voltar uma cópia dele 😀

```suggestion
func getCaixaConfig() *CaixaConfig {
```